src/DependencyInjection/Configuration.php

```diff
@@ -122,7 +123,7 @@ private function addResourcesSection(ArrayNodeDefinition $node): void
                                         ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
                                         ->scalarNode('repository')->defaultValue(BlockRepository::class)->cannotBeEmpty()->end()
                                         ->scalarNode('form')->defaultValue(BlockType::class)->end()
-                                        ->scalarNode('factory')->defaultValue(Factory::class)->end()
+                                        ->scalarNode('factory')->defaultValue(BlockFactory::class)->end()
```

Comment:
```suggestion
                                        ->scalarNode('factory')->defaultValue(Factory::class)->end()
```
Factories are decorated instead